wnt/protocol_buffers_files/message.proto

```diff
@@ -330,11 +352,14 @@ message AppConfigResponse {
         INVALID_REBOOT_DELAY = 18;
         INVALID_DIAG_INTERVAL = 19;
         INVALID_APP_CONFIG = 20;
-        NO_SCRATCHPAD_PRESENT = 21;
-        ACCESS_DENIED = 22;
-        REQUEST_NEEDS_SINK_ID = 23;
-        FIRST_INVALID_ENUM_VALUE = 24;
-        ERROR_UNKNOWN = 255;
+        INVALID_PARAM = 21;
```

Comment:
Hmm, if these come from gateway error codes, is the changed IDs somehow backward compatible?

Reply:
AFAIK, this change was done to split single error code to multiple ones more be more precise, but it will give unfortunately wrong error codes with old gateways. Original gateway API change should have just needed to add new items and not reuse the old ids.